solutions/swb-app/src/datasetRoutes.ts

```diff
@@ -44,9 +47,9 @@ export function setUpDSRoutes(
       processValidatorResult(validate(req.body, CreateDataSetSchema));
```

Comment:
should one of these routes still take `storageName` and `awsAccountId` for external datasets? or will those be different routes?

Reply:
I'd prefer to keep routes separate is it would be a bit tricky to manage + run validation; additionally they are different entities, we might rename this API to reflect that, e.g. `datasets/internal`. @SanketD92 , thoughts ?

Reply:
Initially we were planning to use the `datasets/import` API for external datasets, but it makes sense to separate our APIs for internal and external (the existing ones can stay as-is for internal, no need for rename). For external DS, we should be able to specify the dataset's (non-main) account ID and bucket name (after it has been registered) for importing it as an external DS

Reply:
@bhudnell the answer to your question is based on whether `datasetService.importDataSet()` acts as the entry point for registering both internal and external datasets.